dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/model/ConsumerMethodModel.java

```diff
@@ -53,6 +56,10 @@ public Method getMethod() {
         return method;
     }
 
+    public ConcurrentMap<String, Object> getAttributeMap() {
```

Comment:
Returning the same copy of attributeMap may lead to programatic mistake as many different java file may modify it, we may not have the track of who is modifying what what modifying. 

Should we not have a put and get method of attribute map it self in this class. What do you say?